### ISSUE TITLE
Fix range definitions in reserved fields of TPM2_PT_PCR constants

### ISSUE THREAD

**Issue by tstruk** [tpm2-software/tpm2-tss#793]
Currently they are defined as:
#define TPM2_PT_PCR_RESERVED1    (0x0000000B  0x00000010)
#define TPM2_PT_PCR_RESERVED4    (0x00000017  0x00000210)
which isn't useful.

**Comment by flihp** [comment]
adding this to the 2.0 bugs milestone. Seems like something we want to fix before we ship. @tstruk : Do we know how to handle this one and it's just a matter of putting together a PR or do we have actual work to do here?

**Comment by tstruk** [comment]
I was thinking about adding all the reserved bits, but then I realized that it would be 500+ of #defines, so I think we will just remove them and add a comment that these ranges are reserved.

**Comment by AndreasFuchsSIT** [comment]
I just looked at https://trustedcomputinggroup.org/wp-content/uploads/TSS_Overview_Common_Structures_Version-0.9_Revision-03_Review_030918.pdf

The way the spec works is the following:
```
#define TPMA_NV_RESERVED1_MASK    ((TPMA_NV) 0x00000300)
#define TPMA_NV_RESERVED2_MASK    ((TPMA_NV) 0x01f00000)
#define TPM2_NV_INDEX_INDEX_MASK      ((TPM2_NV_INDEX) 0x00ffffff)
#define TPM2_NV_INDEX_INDEX_SHIFT     (0)
#define TPM2_NV_INDEX_RH_NV_MASK      ((TPM2_NV_INDEX) 0xff000000)
#define TPM2_NV_INDEX_RH_NV_SHIFT     (24)
```

**Comment by tstruk** [comment]
@AndreasFuchsSIT yes, but we need definitions for the TPM2_PT_PCR type.

**Comment by AndreasFuchsSIT** [comment]
Whoops, sorry...
I guess it's a different / new bug then ?

**Comment by tstruk** [comment]
Fixed by 1638939
